src/core/etl/src/Flow/ETL/PHP/Type/Native/ObjectType.php

```diff
@@ -25,9 +25,9 @@ public function __construct(public readonly string $class, private readonly bool
     /**
      * @param class-string $class
      */
-    public static function of(string $class) : self
+    public static function of(string $class, bool $nullable = false) : self
```

Comment:
since we now have a proper ObjectType, ObjectEntry should keep that type internally and pass it to Schema Definition so we can easier validate schema against ObjectEntry just by comparing ObjectTypes.